### ISSUE TITLE
Sorting of data in analysis

### ISSUE THREAD

**Issue by pascalwhoop** [powertac/powertac-tools#14]
Hi John, I referenced a line below. Is the data per row the trades for the target timeslot in column 0 or is it the data of the timeslots that have been traded at the given timestep? 
So if I wanted to get all trades that buy/sell energy FOR the target timeslot x, I can just look into row x (assuming row 1 would be timeslot 1)?

Thanks,

https://github.com/powertac/powertac-tools/blob/78557d54dedb9d9bbd63ba87d1e6f9ad1eb4bedf/logtool-examples/src/main/java/org/powertac/logtool/example/MktPriceStats.java#L44

**Comment by pascalwhoop** [comment]
I thought I'd clarify. Is the row corresponding to the timeslot *in* which the trade occured or the timeslot *for* which the trade occured?

**Comment by pascalwhoop** [comment]
I believe it is the data *for* the timeslot because the image below has the diagonal introduction of values. But I want to clarify. 
![image](https://user-images.githubusercontent.com/2284951/40281422-a9c6bad2-5c61-11e8-8fd7-4e1ac1971137.png)

**Comment by jecollins** [comment]
You are correct, the row corresponding to a given timeslot is the trades for energy delivery in that timeslot. The data is collected from the whole game and stored by target-timeslot in arrays of length 24. So the data for a given timeslot is indexed by leadtime, 0 to 23 hours. Of course, from the standpoint of the broker, that's actually 1 to 24 hours, because orders must be received before the start of a timeslot, which means they are posted by brokers during the previous hour.

I have run this analyzer on a number of games, and often I have seen a slight increase in prices for shorter leadtimes. However, some agents try to learn this pattern and exploit it by buying more than they need at longer leadtimes and then selling the surplus at shorter leadtimes. This tends to flatten the curve, of course.

**Comment by pascalwhoop** [comment]
Interesting so the natural trend actually gets balanced by brokers buying ahead and selling later. Very curious! Thanks for the clarification
